src/scancode/cli.py

```diff
@@ -442,7 +443,14 @@ def scan_event(item):
 
     files_scanned_per_second = round(float(files_count) / scanning_time , 2)
     scan_summary['files_scanned_per_second'] = files_scanned_per_second
-
+    
+    if not scanning_errors:
+        #Log file containing timestamp & Project's name.
+        log = open (input_path + "_log.txt", "a+")
+        day = datetime.datetime.today().replace(microsecond=0)
```

Comment:
tell me what you found there. I am sure I wrote such a function in the past in there